### ISSUE TITLE
Adding Assembly reference leads to error missing reference

### ISSUE THREAD

**Issue by sanjitoktav** [mono/t4#46]
I am trying to build a tt file which has references to other Assemblies.
So I tried to provide the Assembly via command line (-r) which got ignored. Then I changed the assembly to point to the file within the tt file, see below code.

```
<#@ Assembly Name="/root/.nuget/packages/system.data.datasetextensions/4.5.0/lib/netstandard2.0/System.Data.DataSetExtensions.dll" #>
<#@ Import Namespace="System.Data" #>
.... some code
DataSet dsProcedures = new DataSet();		
..... more code
```

But I am getting the error 
> ERROR: The type or namespace name 'DataSet' could not be found (are you missing a using directive or an assembly reference?)

Some more info.
I am using docker with the following Dockerfile
```
FROM microsoft/dotnet:sdk

RUN dotnet tool install -g dotnet-t4
ENV PATH="/root/.dotnet/tools:${PATH}"
```

Before starting the t4 command I am doing a `dotnet restore` on my project so that all dependencies are loaded.

Within Visual Studio I can build the tt files without changing the assembly name so I just have `<#@ Assembly Name="System.Data" #>`

Want we want to achive is a simple ci with a docker container to automate our build.
Maybe this is related to #21 ?

**Comment by mhutch** [comment]
Try adding `-r System.Data` to the invocation

**Comment by mhutch** [comment]
Ah, this was fixed in https://github.com/mono/t4/pull/37. Rolling a new release.

**Comment by mhutch** [comment]
Fixed in 2.0.4.

Verified with 

```bash
echo "<#@ Import Namespace='System.Data' #><# DataSet dsProcedures = new DataSet(); #>" \
| t4 -r System.Data -o -
```

**Comment by sanjitoktav** [comment]
Thanks with 2.0.4 it finds the reference
